README.md

```diff
@@ -2,17 +2,8 @@ This is not a full portage configuration. It contains only those parts that are
 necessary to set up a gentoo github mirror based sync system:
 
 * Configures Portage to sync via https://github.com/gentoo/gentoo.git URL.
-* Updates metadata-cache (the first update might take long, all subsequent ones not).
+* Updates metadata-cache.
 * Updates the dtd directory.
 * Updates the glsa directory.
 * Updates `herds.xml` file.
 * Updates the news directory.
-
-## Notes ##
```

Comment:
Sure no problem, and I see you've already done it. But you could point in the Readme to the cache generation script included here,  portage-gentoo-git-config/etc/portage/repo.postsync.d/sync_gentoo_cache, instead of the "example".